examples/triton_runner/README.md

```diff
@@ -0,0 +1,103 @@
+# Triton Inference Server integration
+
+BentoML now provides support for Triton Inference Server. Triton can be accessed via `bentoml.triton.Runner`:
+
+```python
+triton_runner = bentoml.triton.Runner("local", model_respository="/path/to/model_repository")
+
+svc = bentoml.Service("my-service", runners=[triton_runner])
+```
+
+`model_repository` can also accept S3 path:
+
+```python
+triton_runner = bentoml.triton.Runner("triton-runners", model_repository="s3://path/to/model_repository")
+```
+
+An example of inference API:
+
+```python
+@svc.api(
+    input=bentoml.io.NumpyNdarray.from_sample(
+        np.array([[1, 2, 3, 4]]), enforce_dtype=False
+    ),
+    output=bentoml.io.NumpyNdarray.from_sample(np.array([1])),
+)
+def triton_infer(input_data: NDArray[np.float16]) -> NDArray[np.int16]:
+    iris_res = iris_clf_runner.run(input_data)
+    res_kwargs = triton_runner.txt2img.run(IRIS_NAME=iris_res)
+    return iris_res
+```
+
+Note that each attribute of the `triton_runner` includes the name of all given models under `model_repository`
+
+Add the following to your `bentofile.yaml`:
+
+```yaml
+service: "service.py:svc"
+include:
+  - "*.py"
+  - "model_repository/"
+docker:
+  base_image: nvcr.io/nvidia/tritonserver:22.12-py3
+```
+
+> tritonserver are currently only supported with `--production` tag. Make sure to have `tritonserver` binary available in PATH if running locally.
+
+To pass triton arguments to `serve` do it via `--triton-options ARG=VALUE[, VALUE]`
+
+```bash
+bentoml serve --production --triton-options log-verbose=True
```

Comment:
`--triton-options` should be specific to the Triton runner as opposed to the `serve` command, right? If we have two Triton runners, which one will `--triton-options` affect? Should we put all the Triton options as input arguments to the Triton runner?

Reply:
Currently `--triton-options` only concerns options that can be used by multiple triton runners. Options such as `allow-metrics`, `metrics-port`, `grpc-port` that is specific to each runner are not yet supported.

In the case of BentoML I believe this is fine since the supervisor will manage the gRPC port.

wrt metrics, it is disabled by default for now.

Reply:
Hm, I think this deserves a little more thought; we should definitely support per-runner options.

Can we just add these to the runner config maybe?